src/components/loader-spinner/loader-spinner.test.tsx

```diff
@@ -301,26 +343,22 @@ describe("when custom props are passed", () => {
   });
 
   it.each([1, 2, 3, 5, 10])(
-    "when 'animationTime' is passed as `%s` the animation duration is the correct time",
-    (animationTimes) => {
-      render(<LoaderSpinner animationTime={animationTimes} />);
+    "when 'animationTime' is passed as `%s`, the animation duration is correct",
+    (animationTime) => {
+      mockUseMediaQuery.mockReturnValueOnce(true);
+
+      render(<LoaderSpinner animationTime={animationTime} hasMotion />);
+
       const innerArcSvgElement = screen.getByTestId("inner-arc");
 
       expect(innerArcSvgElement).toHaveStyle(
-        `animation-duration: ${animationTimes}s`,
+        `animation-duration: ${animationTime}s`,
       );
     },
   );
 });
 
 describe("when the component is rendered and the end user prefers reduced motion", () => {
-  beforeEach(() => {
-    const mockUseMediaQuery = useMediaQuery as jest.MockedFunction<
-      typeof useMediaQuery
-    >;
-    mockUseMediaQuery.mockReturnValueOnce(false);
-  });
-
   it("the spinner label renders", () => {
```

Comment:
comment: this is missing a mock, it will always pass as the label will be rendered no matter if the hook is true (and showLabel) or false so we should add another assertion to catch any regressions

```
it("the spinner label renders", () => {
    mockUseMediaQuery.mockReturnValueOnce(false);

    render(<LoaderSpinner />);
    const wrapperElement = screen.getByRole("status");
    const visibleLabelElement = screen.getByTestId("visible-label");
    const hiddenLabelElement = screen.getByTestId("hidden-label");

    expect(wrapperElement).toHaveTextContent("Loading...");
    expect(visibleLabelElement).toBeVisible();
    expect(hiddenLabelElement).not.toBeInTheDocument();
  });